internal/pkg/config/types.go

```diff
@@ -33,65 +35,6 @@ func ListDefaultServices() map[string]string {
 	}
 }
 
-// ServiceInfo contains configuration settings necessary for the basic operation of any EdgeX service.
-type ServiceInfo struct {
-	// BootTimeout indicates, in milliseconds, how long the service will retry connecting to upstream dependencies
-	// before giving up. Default is 30,000.
-	BootTimeout int
-	// Health check interval
-	CheckInterval string
-	// Indicates the interval in milliseconds at which service clients should check for any configuration updates
-	ClientMonitor int
-	// Host is the hostname or IP address of the service.
-	Host string
-	// Port is the HTTP port of the service.
-	Port int
-	// The protocol that should be used to call this service
-	Protocol string
-	// StartupMsg specifies a string to log once service
-	// initialization and startup is completed.
-	StartupMsg string
-	// MaxResultCount specifies the maximum size list supported
-	// in response to REST calls to other services.
-	MaxResultCount int
-	// Timeout specifies a timeout (in milliseconds) for
-	// processing REST calls from other services.
-	Timeout int
-}
-
-// HealthCheck is a URL specifying a healthcheck REST endpoint used by the Registry to determine if the
-// service is available.
-func (s ServiceInfo) HealthCheck() string {
-	hc := fmt.Sprintf("%s://%s:%v%s", s.Protocol, s.Host, s.Port, clients.ApiPingRoute)
-	return hc
-}
-
-// Url provides a way to obtain the full url of the host service for use in initialization or, in some cases,
-// responses to a caller.
-func (s ServiceInfo) Url() string {
-	url := fmt.Sprintf("%s://%s:%v", s.Protocol, s.Host, s.Port)
-	return url
-}
-
-// RegistryInfo defines the type and location (via host/port) of the desired service registry (e.g. Consul, Eureka)
-type RegistryInfo struct {
-	Host string
-	Port int
-	Type string
-}
-
-// LoggingInfo provides basic parameters related to where logs should be written.
-type LoggingInfo struct {
-	EnableRemote bool
-	File         string
-}
-
-// StartupInfo provides the startup timer values which are applied to the StartupTimer created at boot.
-type StartupInfo struct {
-	Duration int
-	Interval int
-}
-
 // MessageQueueInfo provides parameters related to connecting to a message queue
```

Comment:
Why isn't MessageQueueInfo also moved?

Reply:
It isn't referenced by the common bootstrap implementation or its common handlers.